src/components/ImageSearchResultTable.tsx

```diff
@@ -36,6 +40,43 @@ import {
   TableRow,
 } from "./ui/table";
 
+export function tagPill({ value }) {
+  const tag = value ? value.toLowerCase() : "common";
+
+  const raceOptions = ["celestial", "troll", "bug", "monster"];
+  const characterOptions = ["caelum", "elara", "lyra", "tank", "battler"];
+  const fightingOptions = ["dual-wield", "healing", "melee", "ranged"];
```

Comment:
Either way is doable. This was originally more to test color groupings (race, melee, magic).  Alternatively, it could just be a mapping table for tag/color.  That'd probably ultimately be more useful since you'd be able to generate as many color/tag combinations as you'd like.

Reply:
Here is an image of what the pills look like under the colormaps.css usage:
![image](https://github.com/Vandivier/arias-tale-rpg/assets/89026322/0f54616b-ed61-43a2-a58d-cb0fd6ba2801)

Reply:
nice! will merge. let me poke around the font for a few minutes either way this is gtg today